udata/frontend/markdown.py

```diff
@@ -86,9 +86,14 @@ def __call__(self, stream, source_tooltip=False, wrap=True):
         if source_tooltip:
             callbacks.append(source_tooltip_callback)
         
+        allowed_attr_md = current_app.config['MD_ALLOWED_ATTRIBUTES']
+        
+        if current_app.config['MD_ALLOWED_IMG']:
```

Comment:
`MD_ALLOWED_ATTRIBUTES` with img in it is not enough?

Reply:
No because bleach strips it of the `src` tag so the image does not display.

Reply:
I included the title tag which is maybe useless